content/docs/developing-operators/templates.md

```diff
@@ -61,6 +61,17 @@ Sprig functions that allow environment access are disabled. The respective funct
 
 Use actions to provide branching or repetition in templates. Below are some examples on how to use functions in operator templates. The [documentation for Go templates](https://golang.org/pkg/text/template/#hdr-Actions) has more details on its features.
 
+### Perform arithmetic using parameters
+
+Note that all `.Params` have a type of `string`. You may need to use a function to convert a parameter to
+a different type first.
+
+This example substracts 1 from a parameter:
```

Comment:
Should be `subtracts`.

Reply:
thanks @yankcrime !  fixed

Reply:
Wow, apparently I still live [in the XVIII century](https://jakubmarian.com/subtract-subtraction-vs-substract-substraction-in-english/)! I thought this is spelled with an `s` ever since I learned it :sweat_smile: 

> It is also worth noting that the form “substract” did exist in English until about the 18th century (due to French influence), but it is now considered completely obsolete.